apps/workers/src/atlas_workers/workers/ftp_sync/ftp_downloader.py

```
@@ -0,0 +1,175 @@
+        """
+        local_file_path = Path(local_path)
+
+        # Skip if file exists and is not empty
+        if skip_existing and local_file_path.exists() and local_file_path.stat().st_size > 0:
+            logger.info(f"File already exists, skipping: {local_file_path.name}")
+            return True
+
+        # Create directory structure
+        try:
+            local_file_path.parent.mkdir(parents=True, exist_ok=True)
+        except Exception as e:
```

Ruff (0.14.4)

Do not catch blind exception: `Exception`

(BLE001)